### PR TITLE
Update Changelog.md

### PR DESCRIPTION
**Pull request by woensug-choi**
Adding 3.0.0 changes
@crvogt Could you summarize what's happened with Noetic/Gazebo11 support?
@dtdavi1 Could you check if we can solve CI check fails even with the changelog edit?

#### To dos for the release (please add if I am missing something)
- [x] Tag nps_uw_sensors_gazebo repo
- [x] Notify Lindzy of the sonar sensor plugin release and its documents
- [x] Update changelog
- [x] Tag dave repo
- [x] Publish the release
- [x] Notify the community about the release

### CODE REVIEW

**Comment by woensug-choi** [comment]
I'll merge this in separate root and leave it as issues for more updates.

**Comment by woensug-choi** [comment]
@bsb808 Could you review this PR for Changelog? I will tag and make a release.

**Review by bsb808 — APPROVED** [review]
